.github/workflows/sync_contracts.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Sync Chainloop Workflow contracts
+on:
+  push:
+    branches: [ "main" ]
+  schedule:
+    - cron: "0 0 * * *" # daily at midnight
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  diff:
+    name: Sync Chainloop Workflow contracts
+    runs-on: ubuntu-latest
+    env:
+      CHAINLOOP_VERSION: 0.84.0
+      CHAINLOOP_API_TOKEN: ${{ secrets.CHAINLOOP_ROBOT_ACCOUNT_SYNC_CONTRACTS }}
+    steps:
+      - name: Install Chainloop
+        run: |
+          curl -sfL https://raw.githubusercontent.com/chainloop-dev/chainloop/01ad13af08950b7bfbc83569bea207aeb4e1a285/docs/static/install.sh | bash -s -- --version v${{ env.CHAINLOOP_VERSION }}
+
+      - uses: actions/checkout@v4.1.4
+        with:
+          fetch-depth: ${{ github.event_name == 'pull_request' && 2 || 0 }}
+      - name: Get changed files
```

Comment:
I do not think this is what we want. We want to go through the whole directory and apply update always. note that revisions will not bump if the content hasn't changed

Reply:
Regardless of whether the have changed or not?